storage_controller/src/pageserver_client.rs

```diff
@@ -365,6 +403,16 @@ impl PageserverClient {
         )
     }
 
+    #[allow(dead_code)]
```

Comment:
maybe use `#[expect(dead_code)]` so that once we start using these functions we will get a warning to remove the notations?